File: selection_sort/selection_sort.js
const selectionSort = arr => {
    const sorted = []

    while (true) {
        if (arr.length === 0) return sorted

        let biggest = arr[0]
        let biggestIndex = 0

        arr.forEach((el, index) => {
            if (el > biggest) {
                biggest = el
                biggestIndex = index
            }
        })

        arr.splice(biggestIndex, 1)
        sorted.unshift(biggest)
    }
}

const nums = [1, 0, -3, 100, 5, 1, 1, 200, 3, 8000, -200]
const sortednums = selectionSort(nums)
console.log(sortednums)

File: selection_sort/selection_sort.py
nums = [3, 8, 0, -2, 100, -500, 1000, 1]
sorted = []

while len(nums) > 0:
    biggest_index = 0
    biggest = nums[0]

    for i in range(len(nums)):
        if nums[i] > biggest:
            biggest = nums[i]
            biggest_index = i
    
    sorted.insert(0, biggest)
    nums.pop(biggest_index)

print(sorted)
